server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -59,6 +60,10 @@ export class RestGitService {
         private readonly cache?: ICache,
         private readonly asyncLocalStorage?: AsyncLocalStorage<string>,
         private readonly storageName?: string) {
+        if (containsPathTraversal(documentId)) {
```

Comment:
Do you think it would be interesting to move this check to `createGitService()` in utils.ts under routes/? That way we can avoid some imports here, and also keep the RestGitService constructor simpler, transferring the validation responsibility to `createGitService()` (which I guess already does some checking with `parseToken()`)

Reply:
Good idea!